### PR TITLE
Improve overview aa1 to implement 1

### LINKED ISSUE #117: Component Content is Markable
**Describe the bug**
Currently it is possible to mark the content of the overview navbar and the content of badges.

**To Reproduce**
1. Go to any SNEK profile.
2. See that it is possible to mark the navbar content.

**Expected behavior**
The content of the overview navbar and the content of badges should not be markable.

**Screenshots**
Markable navbar:
image

Markable badge:
image

### LINKED ISSUE #118: Activity Chart Re-Caching Bug
**Describe the bug**
Currently the activity charts display the old data after re-caching.

**To Reproduce**
1. Go to login
2. Login with a real SNEK account
3. Wait for re-caching to complete
4. See that the activity charts is still display the old data.

**Expected behavior**
The activity charts should display the latest data.

### LINKED ISSUE #119: Calendar Year Swap Bug
**Describe the bug**
Currently if the year of the calendar is swapped the page is lags and the total contributions don't update.

**To Reproduce**
1. Go to any SNEK profile
2. Click on 2020
3. See that the page lags and the total contributions don't update

**Expected behavior**
By swapping the year of the calendar the page should not lag and the total contributions should fit to the selected year.

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 

 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 

 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the contributions don't change on the 3D calendar if a different year is selected. #119 
- Currently the activity chart does not reload after the re-caching process. #118
- Currently the component content is selectable. #117 
- Currently the languages list can only be on fold . #121  

**What is the new behavior (if this is a feature change)?**
- Now the contributions change on the 3D calendar if a different year is selected. #119 
- Now the activity chart reloads after the re-caching process. #118
- Now the component content is not selectable. #117 
- Now the languages list can be collapsed. #121 

**Other information**:
- 🐍

### CODE REVIEW

File: src/components/atoms/charts/Calendar3D/index.jsx [hunk 1]
```diff
@@ -41,13 +41,26 @@ class Calendar3D extends React.Component {
...
         },
         () => this.checkCache()
       );
+
+      this.checkCache();
     }
   };
 
   componentDidUpdate = () => {
+    if (this.props.year != this.state.year) {
```
Comment (Aichnerc) [inline src/components/atoms/charts/Calendar3D/index.jsx]: Is this shallow comparison necessary? Is `!==` possible?

File: src/components/organisms/SoftwareTabs/softwaretabs.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.nav-tabs {
+  -moz-user-select: none;
+  -webkit-user-select: none;
```
Comment (Aichnerc) [inline src/components/organisms/SoftwareTabs/softwaretabs.scss]: Is moz and webkit required or does SCSS automatically deal with it?

**Review by Aichnerc — APPROVED** [review]
LGTM, found some things though.

File: package.json [hunk 3]
```diff
@@ -8,19 +8,19 @@
     "fuzzysort": "^1.1.4",
     "gh-pages": "^3.1.0",
     "js-sha256": "^0.9.0",
-    "mdbreact": "mdbreact-4.26.1.tgz",
     "mdb-react-sortable": "mdb-react-sortable-4.26.0.tgz",
+    "mdbreact": "mdbreact-4.26.1.tgz",
     "node-sass": "^4.12.0",
     "obelisk.js": "^1.2.2",
-    "react-avatar-editor": "^11.0.7",
+    "react-avatar-editor": "^11.0.9",
```
Comment (schettn) [inline package.json:15]: Why have you upgraded to 11.0.9? I had some issues with this version.
Ref: https://github.com/mosch/react-avatar-editor/issues/340

File: src/components/atoms/charts/LatestActivity/index.jsx [hunk 4]
```diff
@@ -149,6 +168,21 @@ class LatestActivity extends React.Component {
     }
   };
 
+  stringToHash = (string) => {
+    let hash = 0;
+
+    if (string.length == 0) return hash;
+
+    for (let i = 0; i < string.length; i++) {
+      let char = string.charCodeAt(i);
```
Comment (schettn) [inline src/components/atoms/charts/LatestActivity/index.jsx]:
```suggestion
      const char = string.charCodeAt(i);
```

**Review by schettn — CHANGES_REQUESTED** [review]
Have a look

File: package.json (hunk 3 above)
Comment (pinterid) [reply package.json:15]: For me the version 11.0.7 did not work, that's why I upgraded it.

Comment (schettn) [reply package.json:15]: Strange..

Maybe we should fork it and fix it for them.

**Review by schettn — APPROVED** [review]
lgtm

File: src/components/organisms/ProfileInfo/index.jsx [hunk 5]
```diff
@@ -58,7 +60,7 @@ class ProfileInfo extends React.Component {
 
   render() {
     const { fetchedUser } = this.props;
-
+    console.log(fetchedUser.platformData.user);
```
Comment (kleberbaum) [inline src/components/organisms/ProfileInfo/index.jsx]: Please remove this line.
```suggestion

```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/organisms/ProfileInfo/index.jsx [hunk 6]
```diff
@@ -58,7 +60,6 @@ class ProfileInfo extends React.Component {
 
   render() {
     const { fetchedUser } = this.props;
```
Comment (kleberbaum) [inline src/components/organisms/ProfileInfo/index.jsx]:
```suggestion
    const { fetchedUser } = this.props;

```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**